src/runtime/dom-particle.ts

```diff
@@ -56,15 +63,15 @@ export class DomParticle extends XenStateMixin(DomParticleBase) {
   /**
    * Override to return a dictionary to map into the template.
    */
-  render(...args) {
+  render(...args): RenderModel {
     return {};
   }
 
   /**
    * Copy values from `state` into the particle's internal state,
    * triggering an update cycle unless currently updating.
    */
-  setState(state) {
+  setState(state): void {
     return this._setState(state);
```

Comment:
Is return needed for a `:void` function?

Reply:
Good catch. Either this is not `void`, or this shouldn't return.